sklearn/decomposition/pca.py

```diff
@@ -108,11 +108,12 @@ class PCA(_BasePCA):
     """Principal component analysis (PCA)
 
     Linear dimensionality reduction using Singular Value Decomposition of the
-    data to project it to a lower dimensional space.
+    data to project it to a lower dimensional space. 
 
     It uses the LAPACK implementation of the full SVD or a randomized truncated
     SVD by the method of Halko et al. 2009, depending on the shape of the input
-    data and the number of components to extract.
+    data and the number of components to extract. It centers the data matrix
+    columnwise (but does not scale it) before performing Singular Value Decomposition.
```

Comment:
I wouldn't mention the scaling at all; there is no reason why PCA (or any algorithm) would do a preprocessing unless otherwise stated. Maybe,
```
The input data is centered for each feature before applying the SVD. 
``` 
or something similar?
The SVD abbreviation is used the previous sentence, so there is no need to explain what it is.